rxconf/config_types.py

```diff
@@ -1,5 +1,7 @@
 import datetime
 import json
+import hvac  # type: ignore
```

Comment:
Why type ignored?

Reply:
pyright checks did not pass because this library contains lines with missing type hints